R-package/README.md

```diff
@@ -0,0 +1,564 @@
+
+<br>
+
+```R
+
+C:\Program Files (x86)\mingw-w64\i686-7.2.0-posix-dwarf-rt_v5-rev1\mingw32\bin
+
+
+```
+
+<br>
+
+**Installation of cmake**
+
+<br>
+
+First download cmake for Windows, [win64-x64 Installer](https://cmake.org/download/).
+Once the file is downloaded run the **.exe** file and during installation make sure to **add CMake to the system PATH for all users**.
+
+<br>
+
+Before the installation of *rgf* I might have to remove *Rtools* environment variables, such as **C:\\Rtools\\bin** (accessible as explained previously), otherwise errors might occur.
+
+<br>
+
+**Installation of RGF, FastRGF and rgf_python**   [ *assuming the installation takes place in the **c:/** directory* ]
+
+<br>
+
+Open a console with **administrator privileges** (right click on cmd and *run as administrator*), then 
+<br>
+
+
+```R
+
+
+# download the most recent version of rgf-python from the github repository 
+#--------------------------------------------------------------------------
+
+git clone http://github.com/RGF-team/rgf.git
+
+
+
+# then navigate to 
+#-----------------
+
+cd /rgf/RGF/
+
+mkdir bin
+
+cd c:/
+
+
+# then download the latest "rgf.exe" from https://github.com/RGF-team/rgf/releases and place the "rgf.exe" inside the previously created "bin" folder ( /rgf/RGF/bin )
+
+
+
+# installation of RGF
+#--------------------
+
+cd /rgf/RGF/build
+
+mingw32-make
+
+cd c:/
+
+
+
+# installation of FastRGF
+#------------------------
+
+cd /rgf/FastRGF/
+
+mkdir build
+
+cd build
+
+
+# BEFORE PROCEEDING WITH cmake MAKE SURE THAT THE "Rtools" folder IS NOT IN THE SAME DIRECTORY (IF THAT IS THE CASE THEN REMOVE IT TEMPROARILY, i.e. copy-paste the "Rtools" folder somewhere else)
+
+
+cmake .. -G "MinGW Makefiles"
+
+mingw32-make
+
+mingw32-make install
+
+cd c:/
+
+
+# IF APPLICABLE, PASTE THE "Rtools" FOLDER IN THE INITIAL LOCATION / DIRECTORY
+
+
+# installation of rgf-python
+#---------------------------
+
+cd rgf/python-package
+
+python setup.py install
+
+
+```
+
+<br>
+
+Then open a command prompt (console) and type,
+
+<br>
+
+```R
+
+python 
+
+
+```
+
+<br>
+
+to launch *Python* and then type 
+
+<br>
+
+```R
+
+import rgf
+
+exit()
+
+```
+
+<br>
+
+to observe if *rgf* is installed properly. Then continue with the installation of the RGF package,
+
+<br>
+
+```R
+
+install.packages(RGF)
+
+
+```
+
+<br>
+
+On windows the user can take advantage of the RGF package currently **only** from within the command prompt (console). First, find the full path of the installation location of R (possible if someone right-clicks in the R short-cut (probably on Desktop) and navigates to properties >> shortcut >> target). In my OS, for instance, R is located in **C:\\Program Files\\R\\R-3.4.0\\bin\\x64\\R**. Then, by opening a command prompt (console) and giving (for instance in my case),
```

Comment:
Please do something with ` In my OS` and  `in my case` because it's not clear who am I.